docs/tutorials/projects/modernize-retro-robot.md

```diff
@@ -3,10 +3,608 @@ title: "Modernize a 1980s robot"
 linkTitle: "Modernize a 1980s Robot"
 weight: 60
 type: "docs"
-tags: []
-description: ""
+tags: ["base", "retro", "vision", "computer vision", "camera", "motor", "python"]
+description: "Modernize a 1980s robot with Viam - Part 1"
 image: ""
 images: []
```

Comment:
You do still need these. I'd recommend using ezgif.com on the blinking video and turning that into a small gif, which you can then use for the images item and turn it into a video for the preview on the tutorials page. We can also do that for you, of course

Reply:
added